spock-core/src/main/java/org/spockframework/mock/runtime/MockSettings.java

```diff
@@ -0,0 +1,72 @@
+package org.spockframework.mock.runtime;
+public class MockSettings{
+    private java.lang.String name;
```

Comment:
Nullable annotations are missing, or null checks in the constructor/setter missing.